src/i18n/index.tsx

```diff
@@ -1,3 +1,8 @@
+import { useTranslation } from "react-i18next";
+
 export { useTranslation } from "react-i18next";
 export { default as Provider } from "./provider";
 export { default as PublishedProvider } from "./publishedProvider";
+
+// eslint-disable-next-line react-hooks/rules-of-hooks
+export const useT = () => useTranslation().t;
```

Comment:
Is there a reason we need to keep this a function, and not just do `export const t = useTranslation().t;`? 
If we did that we could `import {t} from "@reearth/i18n";` and use t immediately, rather than always declaring useT, cleaning up code more.

Reply:
useTranslation is "a React hook" so it should be always called inside components.